src/main/java/by/a1qa/screens/DiscoverScreen.java

```diff
@@ -13,34 +13,39 @@
 public class DiscoverScreen extends Screen {
     private final String showContainer = "//android.widget.LinearLayout[@index='1']";
     private final String xpathContainer = "//android.view.ViewGroup[@content-desc='Fan Favourites The shows that our audience love']";
-    private final ILabel showsLbl = getElementFactory().getLabel(By.xpath(xpathContainer), "scroll");
-    private final ILabel headerLbl = getElementFactory().getLabel(By.id("com.todaytix.TodayTix:id/discover_header"), "Header lbl");
-    private final By posterNameLoc = By.xpath(showContainer + "//*[contains(@resource-id, 'com.todaytix.TodayTix:id/show_name')]");
-    private final By posterCostLoc = By.xpath(showContainer + "//*[contains(@resource-id, 'com.todaytix.TodayTix:id/price_amount_label')]");
+    private final ILabel showsLbl = getElementFactory().getLabel(By.xpath(xpathContainer), "Shows lbl");
+    private final ILabel header = getElementFactory().getLabel(By.id("discover_header"), "Header lbl");
+    private final By posterLoc = By.xpath(showContainer + "//*[contains(@resource-id, 'container')]");
+    private final By posterNameLoc = By.xpath(showContainer + "//*[contains(@resource-id, 'show_name')]");
+    private final By posterCostLoc = By.xpath(showContainer + "//*[contains(@resource-id, 'price_amount_label')]");
 
     public DiscoverScreen() {
         super(By.xpath("//android.widget.LinearLayout"), "Discover Page");
     }
 
     public boolean isOnDiscoverScreen() {
```

Comment:
why do you need this method?

Reply:
To check if we are on discover screen by unique element